modules/vaos/app/serializers/ccra/referral_list_serializer.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Ccra
+  # Serializer for referral list entries
+  # Used by the ReferralsController to serialize lists of referrals
+  class ReferralListSerializer
+    include JSONAPI::Serializer
+
+    set_type :referrals
```

Comment:
same comment here as with the `ReferralDetailSerializer`

Reply:
Makes sense. Changed.